### PR TITLE
Update ActionMailer Mailer Models and Views documentation [ci skip]

### PR DESCRIPTION
**Pull request by tomkadwill**
### Summary

Improved documentation ActionMailer base.rb

### CODE REVIEW

File: actionmailer/lib/action_mailer/base.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@
 require 'action_mailer/log_subscriber'
 
 module ActionMailer
-  # Action Mailer allows you to send email from your application using a mailer model and views.
+  # Action Mailer allows you to send email from your application using a mailer, model and views.
```
Comment (vipulnsward) [inline actionmailer/lib/action_mailer/base.rb]: This was correct before "Mailer Model"

File: actionmailer/lib/action_mailer/base.rb [hunk 2]
```diff
@@ -84,9 +84,9 @@ module ActionMailer
   # = Mailer views
   #
   # Like Action Controller, each mailer class has a corresponding view directory in which each
-  # method of the class looks for a template with its name.
+  # method of the class looks for a template with it's name.
```
Comment (vipulnsward) [inline actionmailer/lib/action_mailer/base.rb]: This was correct before as well.

File: actionmailer/lib/action_mailer/base.rb [hunk 3]
```diff
@@ -84,9 +84,9 @@ module ActionMailer
...
   #
-  # To define a template to be used with a mailing, create an <tt>.erb</tt> file with the same
+  # To define a template to be used with a mail, create an <tt>.erb</tt> file with the same
```
Comment (vipulnsward) [inline actionmailer/lib/action_mailer/base.rb]: Except this change, can you revert all others, they are correct as is right now.

Comment (vipulnsward) [inline actionmailer/lib/action_mailer/base.rb]: "a mailer"

**Comment by tomkadwill** [comment]
Thanks @vipulnsward. I've updated the PR

**Comment by vipulnsward** [comment]
Thanks @tomkadwill !

**Pull request merged**